showtime-xyz/showtime-frontend
# Mobile improvements (+ Camera)

# Why

We have a new design for the Camera and I found some areas of improvement while testing the app.

# How

- Improve the Camera screen (also improve the code, switching to Tailwind for styling)
- Rename `Discover` to `Marketplace`
- Fix a bug with the Profile screen (probably a merge conflict)
- Improve the routing when clicking on a Media

# Test Plan

Run the app on iOS and Android:
- Click on a profile, it shouldn't crash
- Click on a NFT in the profile, it should open the View NFT modal
- Click on the Camera tab and take a photo
